localstack/services/awslambda/event_source_listeners/dynamodb_event_source_listener.py

```diff
@@ -0,0 +1,72 @@
+import datetime
+from typing import Any, Dict, List, Optional
+
+from localstack.services.awslambda.event_source_listeners.stream_event_source_listener import (
+    StreamEventSourceListener,
+)
+from localstack.services.awslambda.lambda_api import get_event_sources
+from localstack.utils.aws import aws_stack
+from localstack.utils.common import first_char_to_lower
+
+
+class DynamoDBEventSourceListener(StreamEventSourceListener):
+    _FAILURE_PAYLOAD_DETAILS_FIELD_NAME = "DDBStreamBatchInfo"
+
+    @staticmethod
+    def get_source_type() -> Optional[str]:
```

Comment:
It seems like this method was renamed? There is another subclass in ext `EventSourceListenerKafka` which still uses the old signature.